src/components/WebexMeeting/__snapshots__/WebexMeeting.stories.storyshot

```diff
@@ -129,12 +129,20 @@ Array [
               xmlns="http://www.w3.org/2000/svg"
             >
               <path
-                d="M18.0017 20.9998H16.5171C15.6321 21.0217 14.7693 20.7218 14.0886 20.1558C13.4079 19.5898 12.9556 18.7961 12.8157 17.922C13.3101 17.8295 13.7854 17.6542 14.2214 17.4035L13.1019 16.2839C12.6475 16.4667 12.1552 16.5351 11.6682 16.4828C11.1813 16.4306 10.7146 16.2595 10.3094 15.9844C9.90415 15.7094 9.57276 15.3389 9.34441 14.9057C9.11606 14.4724 8.99775 13.9896 8.99992 13.4999V12.1817L7.49992 10.6816V13.4998C7.50176 14.5709 7.88538 15.6063 8.58188 16.4201C9.27839 17.2338 10.2422 17.7726 11.3002 17.9397C11.4415 19.0996 11.9702 20.1779 12.8007 20.9998H6.00175C5.80283 20.9998 5.61207 21.0788 5.47142 21.2195C5.33076 21.3601 5.25175 21.5509 5.25175 21.7498C5.25175 21.9487 5.33076 22.1395 5.47142 22.2801C5.61207 22.4208 5.80283 22.4998 6.00175 22.4998H18.0017C18.2007 22.4998 18.3914 22.4208 18.5321 22.2801C18.6727 22.1395 18.7517 21.9487 18.7517 21.7498C18.7517 21.5509 18.6727 21.3601 18.5321 21.2195C18.3914 21.0788 18.2007 20.9998 18.0017 20.9998Z"
-                fill="var(--wxc-icon--muted-color)"
+                d="M18 21.0002H16.5154C15.6304 21.0221 14.7675 20.7223 14.0868 20.1563C13.4061 19.5903 12.9539 18.7966 12.814 17.9224C13.8476 17.7311 14.7818 17.1843 15.4547 16.3766C16.1276 15.569 16.4969 14.5515 16.4985 13.5002V6.00024C16.4985 4.80677 16.0244 3.66218 15.1805 2.81826C14.3366 1.97435 13.192 1.50024 11.9985 1.50024C10.8051 1.50024 9.66047 1.97435 8.81656 2.81826C7.97264 3.66218 7.49854 4.80677 7.49854 6.00024V13.5002C7.5004 14.5713 7.88398 15.6066 8.58042 16.4204C9.27685 17.2341 10.2405 17.773 11.2985 17.9402C11.4397 19.1001 11.9685 20.1783 12.799 21.0002H6C5.80109 21.0002 5.61032 21.0793 5.46967 21.2199C5.32902 21.3606 5.25 21.5513 5.25 21.7502C5.25 21.9492 5.32902 22.1399 5.46967 22.2806C5.61032 22.4212 5.80109 22.5002 6 22.5002H18C18.1989 22.5002 18.3897 22.4212 18.5303 22.2806C18.671 22.1399 18.75 21.9492 18.75 21.7502C18.75 21.5513 18.671 21.3606 18.5303 21.2199C18.3897 21.0793 18.1989 21.0002 18 21.0002ZM8.99854 11.2412V9.75024H14.9985V11.2412H8.99854ZM11.9985 3.00024C12.7939 3.00104 13.5565 3.31736 14.119 3.8798C14.6814 4.44224 14.9977 5.20484 14.9985 6.00024V8.25024H8.99854V6.00024C8.99933 5.20484 9.31566 4.44224 9.87809 3.8798C10.4405 3.31736 11.2031 3.00104 11.9985 3.00024Z"
+                fill="var(--wxc-icon--color)"
```

Comment:
remove this, the icon should take its color from the parent/context, the svg already has `fill: currentColor` from the `.wxc-icon` class